src/hardware/serialport/nullmodem.h

```diff
@@ -35,30 +35,32 @@
 
 class CNullModem : public CSerial {
 public:
-	CNullModem(Bitu id, CommandLine* cmd);
+	CNullModem(const CNullModem &) = delete;            // prevent copying
```

Comment:
This file's changes are one-to-one type replacements, and member initialization.